data/recommendation/recWindow.html

```diff
@@ -31,8 +31,8 @@ <h3 class="description">description of add-on</h3>
     <div id="recs">
     </div>
     <div id="why-box" hidden>
-      <div class="image-box-info">
-        <img class="image" src="infoIcon.svg">
+      <div class="image-box">
+        <img class="image" id="info-image" src="infoIcon.svg">
```

Comment:
It is generally a bad idea to use IDs for CSS selection. Specifically, IDs must be unique page-wide. It would probably be better to make this `<img class="image info" src="infoIcon.svg">`, and target it as `.info`.